.travis.yml

```diff
@@ -3,8 +3,9 @@ os:
   - linux
   - osx
 julia:
-#  - release # 0.3 not supported yet
-  - nightly
+  - 0.4
+  - 0.5
+# - nightly
```

Comment:
https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail